host/tests/L3_TestCases/dsVideoPort/dsVideoPort_test10_VerifyALLM_Mode.py

```diff
@@ -0,0 +1,112 @@
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+# Get directory path and append to system path
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../../"))
+
+# Import required classes from modules
+from dsVideoPortHelperClass import dsVideoPortHelperClass
+from raft.framework.core.logModule import logModule
+
+class dsVideoPort_test10_VerifyALLM_Mode(dsVideoPortHelperClass):
+    """
+    A class to test and verify Enables/Disables ALLM mode for HDMI output video port.
+
+    Attributes:
+        testName (str): Name of the test.
+        testSetupPath (str): Path to the test setup configuration file.
+        moduleName (str): Name of the module being tested.
+        rackDevice (str): Device under test (DUT).
+    """
+
+    def __init__(self, log:logModule=None):
+        """
+        Initializes the test10_VerifyALLM_Mode test setup and configuration.
+
+        Initializes sessions, reads the test setup, and prepares the user response.
+        """
+        self.testName  = "test10_VerifyALLM_Mode"
+        self.qcID      = '10'
+
+        super().__init__(self.testName, self.qcID, log)
+
+
+    #TODO: Current version supports only manual verification.
+    def testVerifyAllmMode(self, manual=False):
+        """
+        Verifies Enables/Disables ALLM mode for HDMI output video port..
+
+        Args:
+            manual (bool, optional): If True, manual verification is done using user response; otherwise,
+                                     automated verification is used (yet to be implemented).
+
+        Returns:
+            bool: Result of the ALLM mode verification.
+        """
+        if manual == True:
+            #hdcpVersion = self.testdsVideoPort.getHDCPVersion()
+            return self.testUserResponse.getUserYN(f'Is ALLM Mode (Enable/Disable) is displayed on Analyzer (Y/N): ')
+        else :
+            #TODO: Add automation verification methods
+            return False
+
+    def testFunction(self):
+        """
+        Main test function that enables video ports and verifies the ALLM Mode.
+
+        Downloads assets, runs prerequisites, enables/disables ports, and verifies ALLM Mode.
+
+        Returns:
+            bool: Final result of the ALLM Mode verification.
+        """
+
+        # Loop through supported video ports and verify ALLM Mode
+        for port, index in self.testdsVideoPort.getSupportedPorts():
+            self.testEnablePort(port, index)
+            # Set the ALLM Mode as Enable
```

Comment:
could you please add stepstart form here.

Reply:
Actually, all other methods test cases follow a similar approach of adding stepstart . Therefore, I maintained the same format in this test_case for consistency.